ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -317,6 +535,65 @@ export default {
       response: Responses.getOut(),
     },
+    {
+      id: 'DelubrumSav Queen Empyrean Iniquity',
+      netRegex: NetRegexes.startsUsing({ source: 'The Queen', id: '59F9', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Kriegsgöttin', id: '59F9', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Garde-La-Reine', id: '59F9', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'セイブ・ザ・クイーン', id: '59F9', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'DelubrumSav Queen Cleansing Slash',
+      // First hit of Cleansing Slash can be dodged by tank invlunerability
+      // Both first and Second hit, 5BB8, will require tank swaps if damage is taken
+      netRegex: NetRegexes.startsUsing({ source: 'The Queen', id: '59F5' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Kriegsgöttin', id: '59F5' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Garde-La-Reine', id: '59F5' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'セイブ・ザ・クイーン', id: '59F5' }),
+      condition: tankBusterOnParty,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'DelubrumSav Queen Cleansing Slash Doom',
+      // Each Cleansing Slash applies a cleansable Doom (38E), if damage is taken
+      netRegex: NetRegexes.gainsEffect({ source: 'The Queen', effectId: '38E' }),
+      netRegexDe: NetRegexes.gainsEffect({ source: 'Kriegsgöttin', effectId: '38E' }),
+      netRegexFr: NetRegexes.gainsEffect({ source: 'Garde-La-Reine', effectId: '38E' }),
+      netRegexJa: NetRegexes.gainsEffect({ source: 'セイブ・ザ・クイーン', effectId: '38E' }),
+      condition: (data) => data.CanCleanse(),
+      infoText: (data, matches, output) => output.text({ player: data.ShortName(matches.target) }),
+      outputStrings: {
+        text: {
+          en: 'Esuna ${player}',
+          de: 'Medica ${player}',
+          fr: 'Guérison sur ${player}',
+          ko: '"${player}" 에스나',
+        },
+      },
+    },
+    {
+      id: 'DelubrumSav Queen Gods Save The Queen',
+      netRegex: NetRegexes.startsUsing({ source: 'The Queen', id: '59FA', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Kriegsgöttin', id: '59FA', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Garde-La-Reine', id: '59FA', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'セイブ・ザ・クイーン', id: '59FA', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'DelubrumSav Queen Ball Lightning',
+      // Players with Reflect should destroy one for party to stand in the shield left behind
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '7974', capture: false }),
+      suppressSeconds: 1,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Orbs',
```

Comment:
```suggestion
          en: 'Reflect Orbs',
```

(Sorry, I think verbs are helpful here to tell people what to do when there's only one thing to do.)